README.md

```diff
@@ -33,113 +32,60 @@ Please do not copy and paste the commands we provide in this readme without :
 
-This step is only necessary if you want to compile the image yourself. You do not need to do this for the bot to work. This is only if you want to deploy your own image to a server.
-We have a docker hub account, so you may not need to compile the image yourself! You can find it there: https://hub.docker.com/u/draftbot. If you really want to compile it yourself, follow the next step.
+This work is still in progress, please do not open issues about bugs faced in this version of the software.
 
-In the project folder (previously downloaded with git), run:
-
-`docker build . -t draftbot/draftbot`
-
-### Install a database
-
-First, you will need to create a new folder to store the database data. We recommend you to create a folder outside of the project folder, to avoid any issues with git. Copy the path of this folder, you will need in the following command. This is the "path/to/volumes/mariadb"
-
-Create a docker mariadb database with the following command : 
-
-`docker run -d --name mariadb -e MARIADB_USER=draftbot -e MARIADB_PASSWORD=secret_password -e MARIADB_ROOT_PASSWORD=super_secret_password -v /path/to/volumes/mariadb:/var/lib/mysql -p 3306:3306 mariadb:latest`
-
-or with docker compose :
-
-```
-services:
-  mariadb:
-    image: mariadb
-    container_name: mariadb
-    ports:
-      - 3306:3306
-    volumes:
-      - /path/to/volumes/mariadb:/var/lib/mysql
-    environment:
-      MARIADB_USER: draftbot
-      MARIADB_PASSWORD: secret_password
-      MARIADB_ROOT_PASSWORD: super_secret_password
-```
-
-You can also just run a local mariadb server, but we recommend you to use docker. If you want more information about how to install mariadb, you can visit their website : https://mariadb.org/download/
-
-## Deployment
+## Dev environment setup
 
-Building or downloading a docker image of the bot is the recommended way to set up DraftBot in production.
+### Requirements
 
-You need to have a config.toml file filled. The config template can be found at config/config.default.toml
+This list contains all the software programs necessary to run DraftBot. Our start guide will walk you through the installation process
 
-Here is the command that will start the DraftBot docker container:
+- Docker
+- Any IDE
+- git
+- node (see package.json of any service for the relevant version)
+- nvm (not mandatory but highly recommended)
+- yarn
 
-`docker run -d --name draftbot -v /path/to/config.toml:/draftbot/config/config.toml:ro -v /path/to/logs:/draftbot/logs draftbot/draftbot`
+### Start guide
 
-or with docker compose :
+This guide will help you run DraftBot on your computer. We provide instructions for Linux, MacOS (🐧) and Windows (🪟). If not specified, the command is the same for all OS.
+Please do not skip any step of this guide, otherwise, the followings will not work.
```

Comment:
Au lieu de dire "the following" (qui a un "s" en trop, en plus), on pourrait dire "the **instance** won't work", non?